packages/wonder-blocks-tooltip/src/components/tooltip-popper.tsx

```diff
@@ -172,21 +223,39 @@ export default class TooltipPopper extends React.Component<Props> {
     }
 
     render(): React.ReactNode {
-        const {anchorElement, placement} = this.props;
+        const {anchorElement, placement, rootBoundary} = this.props;
+
+        const smallViewportModifier: SmallViewportModifier = {
+            name: "smallViewport",
+            enabled: true,
+            phase: "main",
+            fn: _modifyPosition,
+        };
```

Comment:
**suggestion:** I'd recommend to move this modifier definition at the the file level, that way the `_modifyPosition` function won't be recreated on every render.